src/coreComponents/constitutive/solid/SolidBase.hpp

```diff
@@ -132,6 +132,36 @@ class SolidBaseUpdates
     return 0;
   }
 
+  /**
+   * @brief Get shear modulus
+   * @param[in] k Element index.
+   * @return the shear modulus of element k
+   */
+  GEOSX_HOST_DEVICE
+  virtual real64 getShearModulus( localIndex const k ) const
+  {
+    GEOSX_UNUSED_VAR( k );
+    GEOSX_ERROR( "getShearModulus() not implemented for this model" );
+
+    return 0;
+  }
+
+  /**
+   * @brief Get a parameter representative of the stiffness.
+   * @details This can be used when stabilizations need to be scaled locally by taking into account
+   * the order of magnitude of the stiffness matrix.
+   * @param[in] k Element index.
+   * @return A parameter representative of the stiffness matrix dstress/dstrain
+   */
+  GEOSX_HOST_DEVICE
+  virtual real64 getStiffnessParameter( localIndex const k ) const
```

Comment:
I would remove this function, as the `getShearModulus()` function does nearly what you want.  Having a specific stabilization constant here breaks the API separation between constitutive models and physics solvers.  I would just compute `stiffnessParameter = 2*solid.getShearModulus()` in the assembly kernels (or whatever you need).  If you need other material constants for anisotropic models, though, we can add them here later (or perhaps just use the stiffness matrix directly to compute the appropriate scalings).